pkg/discovery/helper.go

```diff
@@ -74,7 +74,7 @@ type serverResourcesInterface interface {
 }
 
 type helper struct {
```

Comment:
The `pkg/discovery/helper.go` is a porting module from the old version of the client-go package.
Is it possible that we deprecate this module, and use the client-go code instead?

Reply:
Discussed with @blackpiglet offline, the `pkg/discovery/helper.go` calls client-go's discovery code with Velero needed logic wrapped, cannot be replaced by client-go's discovery.